playbooks/install_stack.yaml

```diff
@@ -323,15 +342,15 @@
       tripleo_deploy_deployment_user: stack
       tripleo_deploy_standalone: true
       tripleo_deploy_output_dir: "{{ ansible_env.HOME }}"
-      tripleo_deploy_local_ip: "{{ public_api }}"
+      tripleo_deploy_local_ip: "{{ local_ip }}"
       tripleo_deploy_control_virtual_ip: "{{ control_plane_ip }}"
-      #TODO(emilien) figure out if we need this:
-      #tripleo_deploy_public_virtual_ip: "{{ ssl_enabled | ternary(public_api, omit) }}"
+      tripleo_deploy_public_virtual_ip: "{{ public_api }}"
       tripleo_deploy_environment_files: "{{ default_tripleo_envs + service_envs + tripleo_override_envs }}"
       tripleo_deploy_generate_scripts: true
       tripleo_deploy_keep_running: true
       tripleo_deploy_home_dir: "{{ ansible_env.HOME }}"
       tripleo_deploy_roles_file: "{{ ansible_env.HOME }}/tripleo_standalone_role.yaml"
+      tripleo_deploy_networks_file: /usr/share/openstack-tripleo-heat-templates/network_data_undercloud.yaml
```

Comment:
Is there any risk that network_data_undercloud.yaml evolves in such a way that it's breaking standalone deployments? Shouldn't we ship our own networks file?

Reply:
honestly I don't think it'll break, the Undercloud is pretty rigid, so if it does change we'll maintain our own file at some point, but I think for now we're good.